### ISSUE TITLE
about the fine-tune learning rate is different from the log file

### ISSUE THREAD

**Issue by dreamcubeblock** [qizekun/ReCon#18]
Hi, there is a issue about the fine-tune learning rate, it seems like the actual learning rate of fine-tune in Scanobj-objbg is 2e-4 instead of 2e-5, you can check the log file in your google cloud.

![1723141348443](https://github.com/user-attachments/assets/931eeefe-3044-41df-a8a2-816cacd7769a)
although you set it as 2e-5, it seems like the real learning rate is 2e-4:
![1723141409053](https://github.com/user-attachments/assets/1a752035-bab0-4061-829b-e88b3c6afede)
could you explain why it works and what is the actual learning rate that you set? 
Thank you!

**Comment by dreamcubeblock** [comment]
OK, it looks like you set the learning rate of the encoder and the learning rate of the finetune head to different sizes, can you explain why you did that? Why is this not mentioned in the paper? How did this improve your results? Just want reproduce the result, thank you!

**Comment by qizekun** [comment]
Hi,
Setting different learning rates results in an approximately 0.2% performance improvement, which is a very minor trick and not significant enough to be reported in the paper.

**Comment by dreamcubeblock** [comment]
Thank you for your reply!
